src/components/Card.js

```diff
@@ -1,22 +1,50 @@
+import Api from "./Api";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "5e8acbbd-4426-43e4-895b-01bd99bee13b",
+    "Content-Type": "application/json",
+  },
+});
```

Comment:
Please, delete it

From the 8th sprint it's not allowed to import anything to the classes. Only the `Popup` class can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor, rather than using outside elements or functions.